lib/esm-utils.js

```diff
@@ -3,7 +3,29 @@ const url = require('url');
 
 const formattedImport = async file => {
   if (path.isAbsolute(file)) {
-    return import(url.pathToFileURL(file));
+    try {
+      return await import(url.pathToFileURL(file));
```

Comment:
I don't remember why we have two different calls of `import`. This one and a second one at line 30.
Why do we need this distinction again?
Why is the second `import` outside of `try`?

Reply:
I agree, but I would wait for Mocha to stop supporting Node 10, as Node 10 has the "old" ESM implementation.

Reply:
> Why is the second import outside of try?

I'm also wondering about that.

Reply:
The second import: The other `import` (line 30) is for bare specifiers (e.g. `import 'react'). As I assumed that syntax errors won't happen in packages, I didn't wrap it with try/catch: the idea was to try and minimize the surface of this hack. If you believe we should consolidate the two in the interest of consistency, I will concur.